kernel/src/boot_stage2.rs

```diff
@@ -241,9 +251,26 @@ global_asm!(
         test %esi, %esi
         jz .Lbsp_main
 
-    .Lcheck_command:
-        /* TODO */
-        jmp .Lcheck_command
+        movl ${MAILBOX_VPIDX_ADDR}, %ecx
+
+        /*
+         * The following wait-for-signal code must be written in asm because
+         * APs run stacklessly here.
+         */
+    .Lap_wait_for_signal:
+        /* vCPU index */
+        movl (%ecx), %eax
+        cmp %esi, %eax
+        je 4f
+        pause
+        jmp .Lap_wait_for_signal
```

Comment:
```suggestion
        pause
        jne .Lap_wait_for_signal
```

Same as earlier suggestion.